src/contracts/core/AllocationManagerStorage.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.27;
+
+import "../interfaces/IAVSDirectory.sol";
+import "../interfaces/IAllocationManager.sol";
+import "../interfaces/IDelegationManager.sol";
+import {Snapshots} from "../libraries/Snapshots.sol";
+
+abstract contract AllocationManagerStorage is IAllocationManager {
+    using Snapshots for Snapshots.History;
+
+    /// @notice The EIP-712 typehash for the contract's domain
+    bytes32 public constant DOMAIN_TYPEHASH =
+        keccak256("EIP712Domain(string name,uint256 chainId,address verifyingContract)");
+
+    /// @notice The EIP-712 typehash for the `MagnitudeAdjustments` struct used by the contract
+    bytes32 public constant MAGNITUDE_ADJUSTMENT_TYPEHASH = keccak256(
+        "MagnitudeAdjustments(address operator,MagnitudeAdjustment(address strategy, OperatorSet(address avs, uint32 operatorSetId)[], uint64[] magnitudeDiffs)[],bytes32 salt,uint256 expiry)"
+    );
+
+    /// @notice The DelegationManager contract for EigenLayer
+    IDelegationManager public immutable delegation;
+
+    /// @notice The AVSDirectory contract for EigenLayer
+    IAVSDirectory public immutable avsDirectory;
+
+    /// @notice Delay before deallocations are completable and can be added back into freeMagnitude
+    /// In this window, deallocations still remain slashable by the operatorSet they were allocated to.
+    uint32 public immutable DEALLOCATION_DELAY;
+
+    /// @dev Delay before alloaction delay modifications take effect.
+    uint32 public immutable ALLOCATION_DELAY_CONFIGURATION_DELAY; // QUESTION: 21 days?
+
+    /**
+     * @notice Original EIP-712 Domain separator for this contract.
+     * @dev The domain separator may change in the event of a fork that modifies the ChainID.
+     * Use the getter function `domainSeparator` to get the current domain separator for this contract.
+     */
+    bytes32 internal _DOMAIN_SEPARATOR;
+
+    /// @notice Mapping: operator => salt => Whether the salt has been used or not.
+    mapping(address => mapping(bytes32 => bool)) public operatorSaltIsSpent;
+
+    /// @notice Mapping: operator => strategy => snapshotted totalMagnitude
+    /// Note that totalMagnitude is monotonically decreasing and only gets updated upon slashing
+    mapping(address => mapping(IStrategy => Snapshots.History)) internal _totalMagnitudeUpdate;
+
+    /// @notice Mapping: operator => strategy => operatorSet (encoded) => snapshotted magnitude
+    mapping(address => mapping(IStrategy => mapping(bytes32 => Snapshots.History))) internal _magnitudeUpdate;
+
+    /// @notice Mapping: operator => strategy => OperatorMagnitudeInfo to keep track of info regarding pending magnitude allocations.
+    mapping(address => mapping(IStrategy => OperatorMagnitudeInfo)) public operatorMagnitudeInfo;
```

Comment:
I think we should just call this struct `MagnitudeInfo`

Currently we read like this:

`OperatorMagnitudeInfo storage info = operatorMagnitudeInfo[operator][strategy]`

I think cutting the leading Operator from the struct name will suffice - it's already clear what type of magnitude info it is - not just because of the state variable name, but also because we really only have one type :)

Reply:
Fixed in #786